AWSSDKSwiftCLI/Tests/AWSSDKSwiftCLITests/Commands/GeneratePackageManifestTests.swift

```diff
@@ -16,10 +16,10 @@ class GeneratePackageManifestTests: CLITestCase {
     func createPackageDependencies(
         crtVersion: String,
         clientRuntimeVersion: String
```

Comment:
Below, many tests have been marked with `throws` since creating a `Version` from a string is now a throwing operation.